src/main/java/threads/MyRunnable.java

```diff
@@ -0,0 +1,26 @@
+package threads;
+
+import main.Counter;
+import org.apache.log4j.Logger;
+
+public class MyRunnable implements Runnable {
+    private static final Logger logger = Logger.getLogger(MyRunnable.class);
+    private static final int MAX_INT = 100;
```

Comment:
we have two same constants. let's move this value to Counter class and create a method getMaxNumber() or smth else